### ISSUE TITLE
通道创建的参数项问题，Consortium是否需用户在desktop填入

### ISSUE THREAD

**Issue by dengyi9** [blockchain-desktop/hyperledger-fabric-desktop#222]
导入的configtx.yaml文件，在使用教程中并未提及联盟名称的配置，但是这个参数确实是会写入到通道配置里面的。一般默认情况是“SampleConsortium”但有时候可能不是，这种情况下创建通道就要指定对应的联盟

_Originally posted by @liurain in https://github.com/blockchain-desktop/hyperledger-fabric-desktop/issues/15#issuecomment-492477451_

**Comment by liurain** [comment]
建议将configtx.yaml文件放到后台，根据用户配置的参数去自动修改这个配置，毕竟创建通道时候的配置不是很多，这样可以避免用户去学习configtx.yaml文件的配置

**Comment by dengyi9** [comment]
是的。  如果针对Consortium参数这个问题，按目前设计，由用户负责写好configtx.yaml文件导入，则Consortium不算问题。  

但是更好的体验，是在desktop图形界面输入。 这个思路应该是对应 #84。我也赞同这个做法。 但我们需要确认一下这个文件的各种写法，另外这种配置文件不同fabric版本可能有差异，印象中v1.1和v1.3有区别，我们需要考虑兼容的风险。
